### PR TITLE
feat: update `ContainerRegistry` mutations' response

### PR DESCRIPTION
**Pull request by rapsealk**
This is a follow-up of #1471
This PR enhances `ContainerRegistry`-related mutations to include affected field rather than `{"result": "ok"}`.

**Checklist:** (if applicable)

- [ ] Milestone metadata specifying the target backport version
- [ ] Mention to the original issue
- [ ] Documentation
- [ ] API server-client counterparts (e.g., manager API -> client SDK)
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: src/ai/backend/manager/models/etcd.py [hunk 1]
```diff
@@ -56,6 +56,13 @@ class ContainerRegistry(graphene.ObjectType):
     hostname = graphene.String()
     config = graphene.Field(ContainerRegistryConfig)
 
+    class Meta:
+        interfaces = (graphene.relay.Node,)
+
+    # @classmethod
+    # def get_node(cls, info: graphene.ResolveInfo, id):
+    #     raise NotImplementedError
+
```
Comment (fregataa) [inline src/ai/backend/manager/models/etcd.py:67]: Is this code for graphql node?

**Review by fregataa — COMMENTED** [review]
Looks good!
I am a bit worried that every gql query will request to etcd, which could increase latency.

File: src/ai/backend/manager/models/etcd.py (hunk 1 above)
Comment (rapsealk) [reply src/ai/backend/manager/models/etcd.py:67]: `graphene` requires relay nodes to have `get_node()` class method to be implemented. This method is used to query the object directly by unique id. But yet we do not support this feature, which means it is not necessary.

Please refer: https://docs.graphene-python.org/en/latest/relay/nodes/#nodes

Comment (kyujin-cho) [inline src/ai/backend/manager/models/etcd.py:67]: Please remove commented out code blocks. If you were meant to specify that this feature has to be implemented in near future then please describe about it with `TODO:` tag in front of the comment block.

**Review by kyujin-cho — CHANGES_REQUESTED** [review]

**Pull request merged**